browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -1735,11 +1735,35 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				'vertical': 'true'
 			},
 			{
-				'id': 'data-calculate',
-				'type': 'bigtoolitem',
-				'text': _UNO('.uno:Calculate', 'spreadsheet'),
-				'command': '.uno:Calculate',
-				'accessibility': { focusBack: true,	combination: 'R', de: null }
+				'type': 'container',
+				'children': [ 
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'data-calculate',
+								'type': 'toolitem',
+								'text': _UNO('.uno:Calculate', 'spreadsheet'),
+								'command': '.uno:Calculate',
+								'accessibility': { focusBack: true,	combination: 'R', de: null }
+							}
+						]
+					},
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'data-convert-formula-to-value',
+								'type': 'toolitem',
+								'text': _UNO('.uno:ConvertFormulaToValue', 'spreadsheet'),
+								'command': '.uno:ConvertFormulaToValue',
+								'image': false,
+								'accessibility': { focusBack: true,	combination: 'R', de: null }
+							}
+						]
+					}
+				],
+				'vertical': 'true'
```

Comment:
I would prefer to have a submenu for Recalculate and Formula to Value under the Calculate menu, just like we have in LO, but I would love to hear feedback from @pedropintosilva .  

![Screenshot from 2024-06-01 13-43-23](https://github.com/CollaboraOnline/online/assets/61119120/aed44564-050f-412c-b5af-d7b237df7d2f)